src/orquestra/sdk/_base/_dsl.py

```diff
@@ -745,10 +733,10 @@ class ArtifactFuture:
 
     def __init__(
         self,
-        invocation: orquestra.sdk._base._dsl.TaskInvocation,
+        invocation: orquestra.sdk._base._dsl.TaskInvocation,  # pyright: ignore
         output_index: Optional[int] = None,
         custom_name: Optional[str] = DEFAULT_CUSTOM_NAME,
-        serialization_format: orquestra.sdk._base._dsl.ArtifactFormat = DEFAULT_SERIALIZATION_FORMAT,  # noqa: E501
+        serialization_format: orquestra.sdk._base._dsl.ArtifactFormat = DEFAULT_SERIALIZATION_FORMAT,  # noqa: E501   # pyright: ignore
```

Comment:
**non-blocking**: what's the error raised by pyright here?

Reply:
For all of the answers: pyright doesnt like the _dsl import, which it is right. but we keep the fully-qualified types for docs apperently (at least thats what is stated in the comment)